packages/rocketchat-lib/server/methods/updateMessage.js

```diff
@@ -0,0 +1,36 @@
+Meteor.methods({
```

Comment:
Why did you remove the moment import? We can remove it from `.eslint` globals if you need